internal/operators/validate_schema.go

```diff
@@ -0,0 +1,270 @@
+// Copyright 2022 Juan Pablo Tosso and the OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build !coraza.disabled_operators.validateSchema
+
+package operators
+
+import (
+	"encoding/json"
+	"encoding/xml"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	"github.com/corazawaf/coraza/v3/experimental/plugins/plugintypes"
+	"github.com/kaptinlin/jsonschema"
+	xsdvalidate "github.com/terminalstatic/go-xsd-validate"
```

Comment:
This library uses cgo, we can still use CGO, but CGO must always be a build flag, not part of the default package
cc @jcchavezs 
Also, it doesn't look reliable we should look for alternatives

Reply:
I was looking at the same thing, for the build flag, thanks! I wasn't able to find a better package, but I would absolutely be open to suggestions. One of the considerations in using this package though was fast performance.

Reply:
I believe my comment didn't go through. I was saying since XML usage isn't crucial as we don't seem to have users using them I would just focus on JSON first.

Reply:
Removed XML code and got all of the tests running. Please let me know any additional suggestions